.vscode/launch.json

```diff
@@ -44,6 +44,18 @@
         "--nolazy"
       ],
       "console": "internalConsole",
+    },
+    {
+      "name": "Debug Current TS file",
+      "type": "node",
+      "request": "launch",
+      "runtimeExecutable": "node",
+      "runtimeArgs": [
+        "--nolazy", "-r", "ts-node/register/transpile-only"
+      ],
+      "program": "${file}",
+      "cwd": "${workspaceFolder}",
+      "internalConsoleOptions": "openOnSessionStart",
```

Comment:
This is out of the scope of this PR

Reply:
> This is out of the scope of this PR

Should I revert this commit and update this PR?

Reply:
Or simply add a commit removing the changes at `launch.json`.

Reply:
Done. :)